README.md

```diff
@@ -21,7 +21,7 @@
 
 ## Features
 - Extremely easy deployment
-- [redoc-cli](https://github.com/Redocly/redoc/blob/master/cli/README.md) with ability to bundle your docs into **zero-dependency** HTML file
+- [Redoc-cli](https://github.com/Redocly/redoc/blob/master/cli/README.md) with ability to bundle your docs into **zero-dependency** HTML file
```

Comment:
Not sure about the capitalisation as the project generally uses lowercase for actual packages (as is customary elsewhere) — I'd leave that to staff reviewer eventually.

Reply:
Well, I suggest there could be some sort of terminology guide that guides contributors on the terms used and their correct format as it wasn't stated earlier.

Reply:
Exactly.

I'm not sure where to start, if that's something to be covered under, dunno, contributing, or coc, or somewhere "officially" in the repo wiki or project boards? But having an official reference for styling and branding would help, as it seems that has evolved in the past somewhat.

Reply:
Well, I think it can be covered in a part under the docs under contributing guidelines or on GitHub. Something that enables someone reference easily.

Reply:
There's a WIP style guide here: https://github.com/Redocly/redoc/pull/1346/files#diff-cb328360a4c7e8b7b8073808479a3581 so let's see how that unfolds. Nonetheless it's neither complete in terms of addressing the various forms of product naming nor approved by the maintainers, so it's maybe a place to voice some open questions or feedback what to include in such style guide.